README.md

```diff
@@ -1 +1,127 @@
-# Environment-IQ
+<!-- Readme template from https://github.com/othneildrew/Best-README-Template/blob/master/BLANK_README.md-->
+
+<a name="readme-top"></a>
+
+<!--
+*** I'm using markdown "reference style" links for readability.
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+
+<div align='center'>
+
+<!-- PROJECT SHIELDS -->
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Stargazers][stars-shield]][stars-url]
+[![MIT License][license-shield]][license-url]
+
+</div>
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/Environment-IQ/Environment-IQ">
+    <img src="https://i.imgur.com/mrT2pHx.png" alt="Logo" width="1000" height="80">
+  </a>
+
+<h3 align="center">Environment IQ</h3>
+
+  <p align="center">
+    Visualization tool for environmental data
+    <br />
+</div>
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li><a href="#built-with">Built With</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#authors">Authors</a></li>
+    <li><a href="#acknowledgments">Acknowledgments</a></li>
```

Comment:
this last link doesn't seem to be working, I'm thinking maybe because the spelling of "acknowledgement" is different in the header below (I believe both "acknowledgment" and "acknowledgement" are correct so just need to pick one)

Reply:
Great catch! Thank you Mia. Fixed in my upcoming commit.